src/backend/aspen/app/views/usergroup.py

```diff
@@ -1,17 +1,44 @@
-from flask import jsonify, session
+import json
+from typing import Dict, List, Union
+
+from flask import jsonify, request, Response, session
 
 from aspen.app.app import application, requires_auth
 from aspen.app.views.api_utils import get_usergroup_query
 from aspen.database.connection import session_scope
+from aspen.database.models.usergroup import User
 
 
-@application.route("/api/usergroup", methods=["GET"])
+@application.route("/api/usergroup", methods=["GET", "PUT"])
```

Comment:
I think it probably makes sense to make PUT a separate route from GET.  They don't have enough in commonality to justify a single route.

Reply:
really you think? I thought it was cleaner to have (very basic) get, put, and post in the same view. I don't mind splitting it out though if you think that would be better.